pkg/encryption/encryption.go

```diff
@@ -10,8 +10,12 @@ import (
 	"storj.io/storj/pkg/storj"
 )
 
-// AESGCMNonceSize is the size of an AES-GCM nonce
-const AESGCMNonceSize = 12
+const (
+	// AESGCMNonceSize is the size of an AES-GCM nonce
+	AESGCMNonceSize = 12
+
+	uint32Size = 4
```

Comment:
maybe add a comment here explaining the same thing as in your PR description?

Reply:
Added a short comment